### DISCUSSION TITLE
Updates should work on local git repos as well.

### DISCUSSION THREAD

**Discussion by jwezel** [copier-org/copier#1822] [Ideas]
Updates seem to only work on remote repos. Can't see why it shouldn't work for local ones as well.

**Comment by sisp** [comment]
Can you please elaborate? What's the problem you're facing? Can you provide a reproducible example? It's difficult to help with this little information.

**Comment by jwezel** [comment]
Well, just what I wrote. I created a directory `template`, did `git init` in it, tagged the initial version, created a copier instance with `copier copy template test`, made some changes in `template`, checked them in, tagged the commit, but when I said `copier update test` it said:

`Updating is only supported in git-tracked subprojects.`

**Reply by sisp** [reply]
That's expected because your (generated) project needs to be Git-tracked as well, not only the template project. Copier's update algorithm internally relies on Git (something like 3-way merge), so both the template and the project need to be Git-tracked. If you run

```shell
git init
git add .
git commit
```

in your project prior to running `copier update`, it should work fine.

**Comment by jwezel** [comment]
Oops, true. Apparently the doc didn't make this clear enough.

**Reply by sisp** [reply]
Feel free to submit a PR that improves the docs. We'll be happy to review it. :slightly_smiling_face:
